openshift/templates/pims-routes/app-route.yaml

```diff
@@ -0,0 +1,72 @@
+# This is a deploy template that will setup and configure routes to run the full PIMS solution.
+# Components in template
+#   - Route 1: API
+#   - Route 2: API Swagger docs
+#   - Route 3: APP
+kind: Template
+apiVersion: v1
+metadata:
+  name: pims-api-deploy
+  annotations:
+    openshift.io/display-name: .NET Core 3.1 Deployment Template
+    description: Deployment template for a .NET Core 3.1 application.
+    tags: pims-api,netcore,api
```

Comment:
This should be reflecting of the frontend tech stack etc. It should reference React instead of Net Core. The tags should be updated as well